scripts/install/install_geographiclib.bash

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
This looks good.

The thing is wave_utils (and hence all of libwave) now depends on this niche dependency.

Can you put the new code into a new module (like wave_geography or something)?

Reply:
`pose_cov_comp.hpp` and `angles.hpp` do not rely on GeographicLib. Should I just move `world_frame_conversions.hpp` to wave_geography?

Reply:
Done.